packages/cli-lib/projects.js

```diff
@@ -191,8 +194,39 @@ async function createProject(dest, type, repoName, sourceDir, options = {}) {
   return success;
 }
 
+const createProjectConfig = (folderPath, configOptions = {}) => {
+  const configPath = path.join(folderPath, 'hsproject.json');
+
+  logger.debug(`Creating config file ${configPath}`);
+  return fs.writeFileSync(
+    configPath,
+    JSON.stringify(
+      {
+        label: configOptions.label || 'New project',
```

Comment:
Is the label field required when creating a new project? Similarly, do we allow one account to have two projects with the same label in it? I'm wondering if we should remove the default label here.

Reply:
I added both `label` and `description` with the expectation that they would be available for projects soon. I believe the unique identifier is different from the `label`, but I could be mistaken. I think @mattcoley may have context on this.